### DISCUSSION TITLE
Let only admin import data thru Airlock

### DISCUSSION THREAD

**Discussion by jorpana** [microsoft/AzureTRE#3449] [General]
Is there a way to allow only the admin to import data thru Airlock and disallow user imports? Or someway to import data to user TRE by the admin? Cannot find it from the documentation.

**Comment by marrobi** [comment]
The airlock is typically used by research teams for ad-hoc import and exports. Not by admins - the airlock is about having an approval process - who would approve the import?

Data flows from the data platform by admins are typically carried out using technologies such as azure Data Factory to move data to storage within the workspace, or using private endpoints in a workspace.

Does that help?

**Reply by m1p1h** [reply]
@marrobi - where might I find more detail (e.g. docs or particular code within the project) on suitable approaches for the second scenario you describe here where I want to establish data flows from a data platform to AzureTRE?

The specific scenario I'm thinking of is where a researcher creates a workspace. Can select a particular dataset they would like to pull into a TRE workspace and trigger the transfer. Where as part of this automated process the researcher has the access permissions to pull in such data.

**Reply by marrobi** [reply]
Take a look at https://github.com/UCLH-Foundry/FlowEHR/ and https://github.com/UCLH-Foundry/Data-Access-Request-Seedling

It might be we look at introducing this into the Azure TRE repository in the coming months. It was discussed here - https://github.com/microsoft/AzureTRE/issues/3609
